src/components/Dialog.js

```diff
@@ -73,6 +83,9 @@ Dialog.defaultProps = {
 };
 
 const styles = StyleSheet.create({
+  backdrop: {
+    ...StyleSheet.absoluteFillObject
```

Comment:
I think I made a mistake here using `absoluteFillObject`. I assume, that `flex: 1` would have the same effect. Would you mind testing this?